lib/broadside/cli/gli_commands.rb

```diff
@@ -95,22 +96,23 @@ def add_instance_flag(cmd)
 
 desc 'Deploy your application.'
 command :deploy do |d|
+  add_target_flag(d)
+
   d.desc 'Deploys WITHOUT running predeploy commands'
   d.command :short do |short|
-    add_target_flag(short)
+    add_tag_flag(full)
 
     short.action do |_, options, _|
-      Broadside::Command.deploy_short(options)
+      Broadside::EcsDeploy.new(options[:target]).short
```

Comment:
so ultimately this would look like 
```ruby
Broadside::Deploy.short(options[:target)
```

`Deploy` would basically be the same kind of module as `Command`, but having all the `Deploy` commands in it.  

(or we could just put them all in `Command` given that they'd be basically the same thing; i'm indifferent)